stdlib/src/collections/dict.mojo

```diff
@@ -612,6 +612,30 @@ struct Dict[K: KeyElement, V: CollectionElement](
             return ev.value()[].value
         return None
 
+    fn get(self, key: K) -> Optional[V]:
+        """Get a value from the dictionary by key.
+
+        Args:
+            key: The key to search for in the dictionary.
+
+        Returns:
+            An optional value containing a copy of the value if it was present,
+            otherwise an empty Optional.
+        """
+        return self.find(key)
```

Comment:
**suggestion** `find` is not a Python dict method. See https://docs.python.org/3/library/stdtypes.html#mapping-types-dict . What is does it playing the role of the `get()` method you just added. So maybe, to avoid having two methods doing the same thing, you could rename `find()` into `get()`?